extensions/amp-sticky-ad/1.0/amp-sticky-ad.css

```diff
@@ -58,6 +58,9 @@ amp-sticky-ad[visible] {
   display: block;
 }
 
+.i-amp-sticky-ad-padding-bottom {
+  padding-bottom: 20px;
```

Comment:
As a iPhone SE user, I do not have that ugly bottom bar nor rounded corners.

I think we want `env(safe-area-inset-bottom, 0px)`: https://developer.mozilla.org/en-US/docs/Web/CSS/env

Reply:
@jridgewell iPhone SE would not get this treatment because its aspect ratio is smaller than 2.1. This will exclude all iPhones with a physical home button.

Reply:
My suggestion would fix the hacky aspect ratio check. For devices that do not have a bottom bar, `safe-area-inset-bottom` returns `0`. For devices that do (including future android phones), it'll be the amount to offset by.

Reply:
Also, the iPhone X doesn't always have a bottom bar intersect the content. Only when the root page scrolls (which AMP pages inside a viewer can't do, unfortunately). When the root page is unscrolled, the page itself will have a square bottom for the toolbar (and no offsetting is needed).

Using `safe-area-inset-bottom` fixes both.